pkg/testdata/models/apis/codedeploy/0000-00-00/docs-2.json

```diff
@@ -45,7 +45,7 @@
     "SkipWaitTimeForInstanceTermination": "<p>In a blue/green deployment, overrides any specified wait time and starts terminating instances immediately after the traffic routing is complete.</p>",
```

Comment:
Looks like you're actually finding spelling mistakes in official AWS models. Sounds like we need this tool to run internally, too. 😆